js/queue/queue-dom.js

```diff
@@ -8,29 +8,51 @@ const dequeue = document.querySelector('.btn-take-dequeue');
 const queue = new QueueDataStructure();
 
 const clearQueueInput = () => {
-  // ... your code goes here
+  let inputValue = queueInput.nodeValue
+  queueInput = ""
+  return inputValue
 };
 
 const generateListQueue = () => {
-  // ... your code goes here
+  for (let i = 0; i < queue; i++){
```

Comment:
```suggestion
  for (let i = 0; i < queue.MAX_SIZE; i++){
```
Por esta razón no se estaban mostrando